packages/fast-components-class-name-contracts-msft/src/button/index.ts

```diff
@@ -8,6 +8,6 @@ export interface IMSFTButtonClassNameContract extends IButtonClassNameContract {
     button_outline: string;
     button_lightweight: string;
     button_justified: string;
-    button_span: string;
+    button_textContainer: string;
```

Comment:
This would be a breaking change, lets move this to a release candidate branch for the next major version bump.